packages/devtools_app/lib/src/timeline/flutter/flutter_frames_chart.dart

```diff
@@ -188,32 +189,31 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
       drawBarShadow: false,
       description: desc,
       highLightPerTapEnabled: true,
-      marker: SelectedDataPoint(onSelected: frameSelected),
+      marker: SelectedDataPoint(onSelected: onBarSelected),
       selectionListener: this,
     );
 
     // Compute padding around chart.
     _chartController.setViewPortOffsets(50, 10, 10, 30);
   }
 
-  void frameSelected(int frameIndex) {
-    _controller.frameBasedTimeline.selectFrame(frames[frameIndex]);
+  void onBarSelected(int index) {
+    // TODO(terry): this is a hack. See
+    // https://github.com/flutter/devtools/issues/1381.
+    Timer(
```

Comment:
this is a really unfortunate hack to have to add. 
@DaveShuckerow Any idea why using the Flutter scheduler to schedule the task on idle wasn't working but this is?